dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -0,0 +1,22 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops
+#COMPONENTPREFIX SESN
+#COMPONENTNAMEFOR SESN Session
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+
+# Note, please reserve all messages used here in the com.ibm.ws.session/resources/com/ibm/ws/session/resource/WASSessionCore.nlsprops file
+
+INCORRECT_URI_SYNTAX=SESN0802E: Invalid syntax used by the httpSessionCache URI: {0}.
+INCORRECT_URI_SYNTAX.explanation=The specified httpSessionCache JCache configuration URI utilizes invalid syntax.
```

Comment:
'utilizes' should be replaced with a better term

Reply:
How about "The syntax of the specified httpSessionCache JCache configuration URI is invalid."